src/handle.ml

```diff
@@ -273,6 +273,10 @@ let handle_cmd_aux : sig_state -> command -> sig_state * proof_data option =
             (* Define the binary operator [s]. *)
             let sym, _ = find_sym false ss qid in
             Sign.add_binop ss.signature s (sym, binop); ss
+        | P_config_prover(s)      ->
```

Comment:
Should be done in queries.ml now.

Reply:
Moved in 52e0e894ba15727d9c2b99f1ee3d7965bc15ef0b.